computer_vision/meanthreshold.py

```diff
@@ -0,0 +1,34 @@
+from PIL import Image
+
+"""
+Mean thresholding algorithm for image processing
+https://en.wikipedia.org/wiki/Thresholding_(image_processing)
+"""
+
+
+def mean_threshold(image: Image) -> Image:
+    """
+    image: is a grayscale PIL image object
+    """
+    height, width = image.size
+    mean = 0
+    pixels = image.load()
+    for i in range(width):
+        for j in range(height):
+            pixel = pixels[j, i]
+            mean += pixel
+    mean //= width * height
```

Comment:
```suggestion
    total = sum(sum(pixel for pixel in row) for row in image.load())
    mean = total // (width * height)
```

Reply:
Yes I tried using list builder comprehension initially. However `image.load()` returns a `PixelAccess` object which according to the [documentation](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.load) is not [Iterable](https://stackoverflow.com/a/1111950). Thus when running this code the following exception is thrown:
`TypeError: 'PixelAccess' object is not iterable`